### PR TITLE
Catch flags for Wall, Werror (does not play nice with our arg injecti…

### PR DESCRIPTION
**Pull request by carsonharmon**
…on always)

### CODE REVIEW

File: polybuild/polybuild.py [hunk 1]
```diff
@@ -160,6 +160,8 @@ def poly_build(self, argv) -> bool:
             compile_command.append("-I" + self.meta.compiler_dir + "/../cxx_libs/include/c++/v1/")
             compile_command.append("-L" + self.meta.compiler_dir + "/../cxx_libs/lib/")
         for arg in argv[1:]:
+            if arg == "-Wall" or arg == "-Wextra" or arg == "Wno-unused-parameter" or arg == "-Werror":
```
Comment (woodruffw) [inline polybuild/polybuild.py:163]: Sorry for the drive-by review, but you're missing a `-` before `Wno-unused-parameter` 😉

Comment (carsonharmon) [reply polybuild/polybuild.py:163]: Ayy thanks thanks I missed that :)

Comment (carsonharmon) [reply polybuild/polybuild.py:163]: copy paste issue lol
